### ISSUE TITLE
BIBFRAME in Sinopia issue: instanceOf references incorrect resources.

### ISSUE THREAD

**Issue by gerontakos** [uwlib-cams/rml#38]
bf:Instances have a value of bf:instanceOf that points to an RDA Expression. 
For example:
<https://api.sinopia.io/resource/1e6cfa13-1dba-44c7-b31d-cc7c0175a380> bf:instanceOf <https://api.sinopia.io/resource/2db1659f-9aa2-4315-b4aa-b00a69c4f3f2>.
It is probably an instance of https://api.sinopia.io/resource/340a8a08-3546-4ff5-874e-d430162c576e 

This was also true for the bf:Instance  https://api.sinopia.io/resource/3934ee78-8555-4fca-bf15-192a3e92b1bf
...which claims to be an instanceOf https://api.sinopia.io/resource/a437ddab-f9c3-47b5-9acc-e20802c71e41 (an RDA Expression)

**Comment by mcm104** [comment]
The second one is a weird one -- <https://api.sinopia.io/resource/a437ddab-f9c3-47b5-9acc-e20802c71e41> seems to have gone missing! I realized it was missing from my most recent RDA folders, and when I ctrl+F in https://api.sinopia.io/resource?limit=1000&group=washington (where I pull all of our IRIs from) for `"uri":"https://api.sinopia.io/resource/a437ddab-f9c3-47b5-9acc-e20802c71e41"`, I get nothing! It doesn't get into my RDA folder, it doesn't get transformed into BIBFRAME, and then there's no BIBFRAME IRI to put in its place!

**Comment by gerontakos** [comment]
We may have a world of hurt here. I'm not sure. I'll give you some clues @mcm104 and see what you can see. I hope it's not a time-consuming heap of nothing. But you'll need to investigate.

When I look at the repo [output](https://github.com/uwlib-cams/rml/tree/master/output) folder, I see https://api.sinopia.io/resource/a437ddab-f9c3-47b5-9acc-e20802c71e41  in the May 4 expressions but not in the May 25 expressions. When I compare all the expressions, May 4 vs. May 25, I see over 30 that are in May 4 but not May 25; for example the following uuids are in May 4 (not May 25):
02c8be46-235a-4bbc-b18d-85065829236d
02fc5cda-9c05-4cf1-b634-382f76522b57
0e468b37-033d-4ebe-85cf-5cdad4da9abd
0fdb4413-f1ca-4f14-b60b-06156c4e0c8d
19e8abd4-f1e4-47b6-afa0-7da404615a17
27a9720d-4d6b-4656-bf50-2ed39776d38a
3b3a6019-a83b-434d-926f-135078efde5e
41b29c94-25c8-4566-9287-34c3d5fc1809

Maybe that's nothing. However if I look for one that's in both and one that's in May 4 only, what do I find? Let's see.

**Case 1: rda:Expression in both the github repo May 4 and May 25 input folders:**
**https://api.sinopia.io/resource/1225f60f-601a-4828-b0a2-5d406e9343c2**
_Is this rda:Expression in Sinopia_? Yes. creationDate=2020-05-04; changeDate=NULL.
_Does this rda:Expression have a triple owl:sameAs pointing to a bf:Work?_ No.
_When I use search criteria (rdae:hasWorkExpressed="Hillbilly Sue") to find the BIBFRAME, does a bf:Work appear?_ No, there's only one work, https://api.sinopia.io/resource/7c057dde-acbe-4298-ac86-78827e4aed72, for Hillbilly Sue, and is is the sameAs https://api.sinopia.io/resource/94c8153f-01b0-42ad-b356-70f3bd64e6dc; however this IRI is not referenced in the rda:Expression as hasWorkExpressed because the value of that property for the rda:Expression is the literal "Hillbilly Sue."
_If the bf:Work does appear in a search, does it state that is is bf:expressionOf the correct bf:Work?_ Not applicable.
_Note:_ Okay, not what I expected. I expected the bf:Work derived from the rda:Expression to be present; it wasn't.

**Case 2: rda:Expression in the github repo May 4 only:**
**https://api.sinopia.io/resource/02c8be46-235a-4bbc-b18d-85065829236d**
_Is this rda:Expression in Sinopia?_ Yes. creationDate=2020-06-10; changeDate=NULL
_Does this rda:Expression have a triple owl:sameAs pointing to a bf:Work?_ No.
_When I use search criteria (hasWorkExpressed=<https://api.sinopia.io/resource/b7ef5c30-4e4c-459c-8361-2c371359894f>; title of that work is "Help-seeking patterns and perceived barriers to care among Latino immigrant men with unhealthy alcohol use" so my search criteria are "Help-seeking patterns") to find the BIBFRAME, does a bf:Work appear?_  Yes, two works, but they are both owl:sameAs <https://api.sinopia.io/resource/b7ef5c30-4e4c-459c-8361-2c371359894f> and neither use the property bf:expressionOf!
_If the bf:Work does appear in a search, does it state that is is bf:expressionOf the correct bf:Work?_  No.
Note: Again not what I expected, but serious errors appear.

What next? Do we need a meeting?

**Comment by mcm104** [comment]
Weirdly, the problem seems to have fixed itself ??

I created a list of records that were in 5/4 but missing from 5/25, and started to look for those in Sinopia here <https://api.sinopia.io/resource?limit=1000&group=washington> expecting them to be missing as well, but I was finding every single one -- and then I looked again for <https://api.sinopia.io/resource/a437ddab-f9c3-47b5-9acc-e20802c71e41> which was missing when I searched 5/25, and it's back! I just pulled all RDA records again using my script without making any changes to it, and they're all there now -- there's nothing in 5/4 that's missing from my new 6/21 folder.

So ??? I have no idea why these records were missing between 5/4 and 5/25, but they're back so... crisis averted...? I'll run the transform in its entirety so these no-longer-missing records get converted.
